fastq_to_TMB_processes.nf

```diff
@@ -8,104 +8,111 @@ snpSift="java -jar /usr/TMB/snpEff/SnpSift.jar"
 process copy_reference {
 	tag "$fasta_name"
 	input:
-		val(fasta_name) 
+		val(fasta_name)
 
 	output:
-		path("*.fa") 
+		path("*.fa")
 
 	script:
-	"""
+		"""
```

Comment:
Looks like the spacing for the script section is still off in some processes.   Probably because I used multiple editors during dev, so maybe some tabs snuck in and differed.

Reply:
Looks like there's tabs in some places and spaces in others. I'm going to make it consistent across all the files.